Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:latest
+
+LABEL org.opencontainers.image.source https://github.com/mlteamurfu2325/practicum-s1
```

Comment:
Apart from adding `LABEL` to link our image to repo, I think we should also add this from GHCR.io's docs:
```
LABEL org.opencontainers.image.description DESCRIPTION
```